components/SearchResult.tsx

```diff
@@ -43,9 +48,19 @@ export default function SearchResult({ searchValue }: { searchValue: string }) {
 
   return (
     <>
-      {data?.results.map((album: Album) => (
-        <Result key={album.id} album={album} />
+      {data?.pages.map((page) => (
+        <>
+          {page.results.map((album: Album) => (
+            <Result key={album.id} album={album} />
+          ))}
+        </>
       ))}
+      <button
```

Comment:
you also only want to show this button if there are more results to show (look at the [Discog's pagination docs](https://www.discogs.com/developers#page:home,header:home-pagination) on how to determine this)